src/bucket.ts

```diff
@@ -273,7 +273,7 @@ export class S3Bucket {
       versionId: res.headers.get("x-amz-version-id") ?? undefined,
       websiteRedirectLocation:
         res.headers.get("x-amz-website-redirect-location") ?? undefined,
-      meta: Object.keys(meta).length !== 0 ? meta : undefined,
+      meta: Object.keys(meta).length !== 0 ? meta : {},
```

Comment:
Is the ternary even needed here? Doesn't `Object.keys(meta).length == 0` mean that it is a `{}`?

Reply:
very true